README.md

```diff
@@ -27,15 +27,14 @@ of the current project.
 
  Our CI pipeline builds and tests on following `x86-64` platforms:
 
-* ubuntu-18.04 ( Default GCC Compiler - 7.5.0 )
-* ubuntu-18.04 ( GCC 4.8 with -std=c++11 flag )
-* ubuntu-20.04 ( Default GCC Compiler - 9.3.0 with -std=c++20 flags )
-* macOS 10.15 ( Xcode 12.2 )
-* Windows Server 2019 (Visual Studio Enterprise 2019 )
+* ubuntu-18.04 (Default GCC Compiler - 7.5.0)
+* ubuntu-18.04 (GCC 4.8 with -std=c++11 flag)
+* ubuntu-20.04 (Default GCC Compiler - 9.3.0 with -std=c++20 flags)
+* macOS 10.15 (Xcode 12.2 )
```

Comment:
nit: we can remove spacing here too after 12.2 ?

Reply:
Sorry I missed it. I will send another PR enforcing markdownlint in the CI and also fixing this nit.

Reply:
Sent the follow up PR #578.